docs/source/onboarding_resources/tutorial/index.ipynb

```diff
@@ -2,7 +2,17 @@
  "cells": [
```

Comment:
<p>"7 days" might be clearer for some readers than "1 week"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='96'/>